assets/src/apps/authoring/components/PropertyEditor/schemas/lesson.ts

```diff
@@ -1,49 +1,56 @@
 import CustomFieldTemplate from '../custom/CustomFieldTemplate';
+
 const lessonSchema = {
   type: 'object',
   properties: {
+    title: {
+      type: 'string',
+      title: 'Title',
+    },
     Size: {
-      type: "object",
-      title: "Dimensions",
+      type: 'object',
+      title: 'Dimensions',
       properties: {
         width: { type: 'number' },
-        height: { type: 'number' }
-      }
+        height: { type: 'number' },
+      },
     },
     Appearance: {
       type: 'object',
       title: 'Lesson Appearance',
       properties: {
         theme: {
-          type:'string',
+          type: 'string',
```

Comment:
good to see these changes :+1: 
interesting that they weren't in place already, maybe it would make sense to add
https://github.com/prettier/eslint-plugin-prettier#recommended-configuration
to the eslint config to allow for automatic fixing of prettier through eslint and allow warning on CI if prettier styles aren't applied?

Reply:
created #1256 to follow up on